caipyra-2016/videos/shu-conhecendo-os-fundamentos-da-linguagem-armando-neto.json

```diff
@@ -0,0 +1,15 @@
+{
+  "description": "Anota\u00e7\u00f5es da palestra: http://piratepad.net/caipyra-armando\nhttp://caipyra.python.org.br",
```

Comment:
http://piratepad.net/caipyra-armando seems empty

Reply:
Yep! Armando did not use slides that day, and the piratepad went blank shortly after it

Reply:
Done